lib/OpenAPI/Client.pm

```diff
@@ -199,12 +198,13 @@ sub _param_as_array {
 
 sub _url_to_class {
   my ($self, $package) = @_;
+  $self = ref $self if ref $self;
```

Comment:
I'm not sure if we should support calling new() on an object.

Reply:
> Should it die, or should it just crash when it naturally would?

Just let it naturally fail. I don't think this is common enough case to add error handling for.